lib/tasks/assets.rake

```diff
@@ -10,9 +10,32 @@ namespace :assets do
     BookmarkletRenderer.compile
   end
 
+  desc "Create non digest assets"
+  task non_digest_assets: :environment do
+    logger = ::Logging::Logger["assets:non_digest_assets"]
+
+    non_digest_assets = Diaspora::Application.config.assets.non_digest_assets
+    manifest_path = Dir.glob(File.join(Rails.root, "public/assets/manifest-*.json")).first
```

Comment:
I only have one file, and it gets updated when I do a precompile. If somebody has two files, it's probably a problem anyway, because sprocket also simply takes the first file: https://github.com/rails/sprockets/blob/v2.12.4/lib/sprockets/manifest.rb#L47-L55